config/locales/devise.en.yml

```diff
@@ -52,7 +52,7 @@ en:
       updated: "Your account has been updated successfully."
       updated_but_not_signed_in: "Your account has been updated successfully, but since your password was changed, you need to sign in again"
     sessions:
-      signed_in: "Signed in successfully."
+      signed_in: ''
```

Comment:
Why do we still need to define this?

Reply:
Devise still calls it, I'm not aware of a mechanism to tell Devise not to display the message.

Reply:
👍